velox/exec/StreamingAggregation.cpp

```diff
@@ -221,11 +202,17 @@ const SelectivityVector& StreamingAggregation::getSelectivityVector(
   return rows ? *rows : inputRows_;
 }
 
-void StreamingAggregation::evaluateAggregates() {
+void StreamingAggregation::evaluateAggregates(
+    const std::vector<vector_size_t>& newGroups) {
```

Comment:
This parameter is no longer used. Let's remove.

Reply:
Done, sorry that I forgot to remove it.